data/data-pipeline/data_pipeline/etl/score/schemas/datasets.py

```diff
@@ -37,30 +37,46 @@ class LoadField:
                 short_name (str): Used in conjunction with the dataset's `short_name` for files where short names are needed.
                 df_field_name (str): Name for the field in the etl class.
                 long_name (str): Column name for the dataset's output csv.
-                field_type (FieldType): An enum that dictates what type of field this is. This will be used on the `etl_score_post`
-                for the data manipulation. The `by_value` metadata prop will load the field type's Enum value instead of the index, i.e. "string"
-                and not STRING reverse_percentile (Optional bool): An optional bool to denote this field to be a reverse_percentile.
-                TODO: data/data-pipeline/data_pipeline/etl/score/etl_score.py:477
+                field_type (FieldType): An enum that dictates what type of field this is. This will be used on the `etl_score_post` for the
+                    data manipulation. The `by_value` metadata prop will load the field type's Enum value instead of the index, i.e. "string"
+                    and not STRING
+                number_of_decimals_in_output (int): Used to represent number of decimals in side effects, like Excel.
                 include_in_tiles (bool): Include this field on the tile export.
                 include_in_csv (bool): Include this field on the CSV export.
                 include_in_excel (bool): Include this field on the Excel export.
-                column_in_position (int): column position in CSV and Excel files
+                column_position (float): column position in CSV and Excel files. Value can be 2.5 to include an item between 2 and 3.
+                create_percentile (bool): Whether or not the backend processing should create a percentile field (ranked in ascending order) from the
```

Comment:
love all these!

Reply:
Could you note that the default behavior (when the param is not specified) is equivalent to `False`?